metadata_service/proxy/gremlin_proxy.py

```diff
@@ -1151,7 +1152,7 @@ def _get_table_columns(self, *, table_uri: str) -> List[Column]:
             col = Column(name=_safe_get(result, 'column', 'name'),
                          key=_safe_get(result, 'column', self.key_property_name),
                          description=_safe_get(result, 'description', 'description'),
-                         col_type=_safe_get(result, 'column', 'col_type'),
+                         col_type=_safe_get(result, 'column', 'type'),
```

Comment:
From: https://github.com/amundsen-io/amundsendatabuilder/blob/master/databuilder/models/table_metadata.py#L160

Reply:
we could change it databuilder actually to make it compatible. In fact, we should consider how we could build the model based on common repo. cc @allisonsuarez